lib/storage/client.js

```diff
@@ -1055,10 +1055,11 @@ class Client extends CryptoConsumer {
    *
    * @param {string} name  the name of the group
    * @param {array} capabilities an array of strings of the capabilities the group will have
+   * @param {string} description  details about the group
    *
    * @returns {Group} A response from TozStore; the group that has been written.
    */
-  async createGroup(name, capabilities = []) {
+  async createGroup(name, capabilities = [], description) {
```

Comment:
We might want description to have a sane default. either an empty string or "A TozStore group"